_data/toc.yaml

```diff
@@ -1330,6 +1330,8 @@ manuals:
     title: Docker Scan
   - path: /engine/sbom/
     title: Docker SBOM (Experimental)
+  - path: /engine/containerd/
+    title: containerd integration (Experimental)
```

Comment:
Do we want this section at the top  level, or under `Desktop` ? (As it only ships with Docker Desktop, so effectively is an experimental feature of Docker Desktop).

Reply:
I am going to be discussing some of these details with Allie tomorrow after our meeting/alignment in the morning.

Reply:
@thaJeztah, @sam-thibault if we have nothing else for this page, can we then approve? @aevesdocker, I think we're ready for your review.